src/test/java/com/browserstack/test/suites/login/LoginFailTest.java

```diff
@@ -0,0 +1,21 @@
+package com.browserstack.test.suites.login;
+
+import com.browserstack.test.suites.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginFailTest extends TestBase {
+
+    @Test
+    public void loginSuccess() {
+        getDriver().findElement(By.id("signin")).click();
```

Comment:
can we have the "driver" object available as-is in this file rather than multiple getDriver() calls?